src/acceptance/java/io/codurance/pouch/DatabaseHelper.java

```diff
@@ -14,6 +14,7 @@
         private final Instant added;
         private final String title;
         private final String url;
+        private boolean favourite;
 
         Resource(UUID id, Instant added, String title, String url) {
```

Comment:
Although for bool false is the default, I like to make it explicit on the constructor.
this.favourite = false

Reply:
Good point!

Reply:
There's an argument to say that if you need to mutate the object - you should create a new one.
Immutable objects make it much easier to reason with systems.
EG: My code senses are asking - why isn't `favourite` declared `final`?